src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/OasysService.kt

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.api.model.OffenceDetail
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.oasysApi.OasysApiService
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.oasysApi.model.OasysOffenceDetail
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.common.exception.NotFoundException
+
+@Service
+class OasysService(val oasysApiService: OasysApiService) {
```

Comment:
I'm wondering if this being a service calling another very similarly named service might cause a bit of confusion with how we've structured clients and services elsewhere, where we tend to have a service taking a client and calling that client's methods. Is there another name/pattern we could use for this type of class? We've used `*Transformer` for other data-manipulating classes in the past, is that something we could use here?